README.md

```diff
@@ -3,90 +3,107 @@
   <img src=".github/wb-logo-darkbg.png#gh-dark-mode-only" width="600" alt="Weights & Biases"/>
 </p>
 
-# Weights and Biases [![PyPI](https://img.shields.io/pypi/v/wandb)](https://pypi.python.org/pypi/wandb) [![Conda (channel only)](https://img.shields.io/conda/vn/conda-forge/wandb)](https://anaconda.org/conda-forge/wandb) [![CircleCI](https://img.shields.io/circleci/build/github/wandb/wandb/main)](https://circleci.com/gh/wandb/wandb) [![Codecov](https://img.shields.io/codecov/c/gh/wandb/wandb)](https://codecov.io/gh/wandb/wandb)
+<p align='center'>
+<a href="https://pypi.python.org/pypi/wandb"><img src="https://img.shields.io/pypi/v/wandb" /></a>
+<a href="https://anaconda.org/conda-forge/wandb"><img src="https://img.shields.io/conda/vn/conda-forge/wandb" /></a>
+<a href="hhttps://circleci.com/gh/wandb/wandb"><img src="https://img.shields.io/circleci/build/github/wandb/wandb/main" /></a>
+<a href="https://codecov.io/gh/wandb/wandb"><img src="https://img.shields.io/codecov/c/gh/wandb/wandb" /></a>
+</p>
+<p align='center'>
+<a href="https://colab.research.google.com/github/wandb/examples/blob/master/colabs/intro/Intro_to_Weights_%26_Biases.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" /></a>
+</p>
 
 Use W&B to build better models faster. Track and visualize all the pieces of your machine learning pipeline, from datasets to production models.
 
-- Quickly identify model regressions. Use W&B to visualize results in real time, all in a central dashboard.
-- Focus on the interesting ML. Spend less time manually tracking results in spreadsheets and text files.
-- Capture dataset versions with W&B Artifacts to identify how changing data affects your resulting models.
-- Reproduce any model, with saved code, hyperparameters, launch commands, input data, and resulting model weights.
+<p align='center'>
```

Comment:
There are placeholder images, waiting to hear from design if they have higher quality assets and assets w/ clear backgrounds.

Reply:
still, love how these look! in case things stay as is, here's a tiny size mismatch: 
<img width="448" alt="image" src="https://user-images.githubusercontent.com/7557205/210479501-6bad8ab0-1f70-441a-9bcd-dd861c214bc9.png">